ofrak_core/ofrak/cli/command/gui.py

```diff
@@ -31,9 +31,14 @@ def create_parser(self, ofrak_subparsers):
             help="Set GUI server host port.",
             default=8080,
         )
+        gui_parser.add_argument(
```

Comment:
Add corresponding arguments to `unpack` and `identify` (GUI arguments are duplicated there for now, perhaps a future refactor can just nest all of these GUI arguments in those commands' arguments)